mmocr/models/ner/classifer/ner_classifier.py

```diff
@@ -0,0 +1,56 @@
+from mmdet.models.builder import DETECTORS, build_loss
+from mmocr.models.builder import build_convertor, build_decoder, build_encoder
+from mmocr.models.textrecog.recognizer.base import BaseRecognizer
+
+
+@DETECTORS.register_module()
+class NerClassifier(BaseRecognizer):
+    """Base class for NER classifier."""
+
+    def __init__(self,
+                 encoder=None,
+                 decoder=None,
+                 loss=None,
+                 label_convertor=None,
+                 train_cfg=None,
+                 test_cfg=None,
+                 pretrained=None):
+        super().__init__()
+        assert label_convertor is not None
```

Comment:
You can make encoder/decoder/loss/label_convertor positional arguments. And the four assersions can be removed.